src/shared/utils.ts

```diff
@@ -500,15 +505,18 @@ export function toast(text: string, background = "success") {
   }
 }
 
-export function pictrsDeleteToast(
-  clickToDeleteText: string,
-  deletePictureText: string,
-  failedDeletePictureText: string,
-  deleteUrl: string
-) {
+export function pictrsDeleteToast(filename: string, deleteUrl: string) {
   if (isBrowser()) {
-    let backgroundColor = `var(--light)`;
-    let toast = Toastify({
+    const clickToDeleteText = `${i18n.t(
```

Comment:
Since these translation keys always use the filename, I think it would more sense to have the filename be an argument passed to the translation instead of using the template strings.